### ISSUE TITLE
How about the result of the implementation?

### ISSUE THREAD

**Issue by Sarah20187** [LinHungShi/GCNetwork#2]
Could you tell me that can this implementation achieves the result authors shown in GC-NET paper?

**Comment by LinHungShi** [comment]
Hi, I haven't reproduced the result because the the whole training set in SceneFlow is too large.  I am still trying to figure out why the author needs only two days to complete training. By the way, the one  you forked is the older version, which is just a trial version. I will update the repository in a few days and write instructions and tips for running the model :)

**Comment by Sarah20187** [comment]
Got it. Thank you! <3

**Comment by wuleibupt** [comment]
Could you realize the results mentioned in the paper ? I  needs 7 days to complete training and the loss can not reduce to 10 or less.

**Comment by LinHungShi** [comment]
@xghc1991  which data you trained with ? I am only using driving data in SceneFlow dataset.

**Comment by Sarah20187** [comment]
@LinHungShi You mean KITTI2015‘s stereo dataset, the newest one?

**Comment by LinHungShi** [comment]
No, I am using data from here. 
https://lmb.informatik.uni-freiburg.de/resources/datasets/SceneFlowDatasets.en.html

**Comment by Sarah20187** [comment]
OK. That's a big data set.

**Comment by wuleibupt** [comment]
We trained the network on Scene Flow dataset, which contains 35454 training and 4370 testing images of size 960 ×540px from an array of synthetic scenes. Now we can reduce loss to 2.5 or less. But sometimes, loss can become big. Now we have not trained kITTI datasets yet


The iteration:  85080 The left loss:  3.268    The right loss:  2.17675
The iteration:  85081 The left loss:  0.744474    The right loss:  0.858949
The iteration:  85082 The left loss:  0.596367    The right loss:  0.651131
The iteration:  85083 The left loss:  0.627725    The right loss:  0.996176
The iteration:  85084 The left loss:  9.87015    The right loss:  8.65417
The iteration:  85085 The left loss:  4.2139    The right loss:  4.03663
The iteration:  85086 The left loss:  2.94178    The right loss:  1.49229
The iteration:  85087 The left loss:  0.661045    The right loss:  0.679529
The iteration:  85088 The left loss:  3.92712    The right loss:  5.2216
The iteration:  85089 The left loss:  1.16924    The right loss:  1.15281
The iteration:  85090 The left loss:  1.60492    The right loss:  12.1845
The iteration:  85091 The left loss:  1.04315    The right loss:  0.549244
The iteration:  85092 The left loss:  1.09168    The right loss:  1.1229
The iteration:  85093 The left loss:  1.93969    The right loss:  0.911412
The iteration:  85094 The left loss:  1.92985    The right loss:  2.2167
The iteration:  85095 The left loss:  0.401371    The right loss:  0.388804
The iteration:  85096 The left loss:  5.39381    The right loss:  3.48623
The iteration:  85097 The left loss:  0.412143    The right loss:  0.69911
The iteration:  85098 The left loss:  0.439725    The right loss:  1.12458
The iteration:  85099 The left loss:  0.757266    The right loss:  0.955459
The iteration:  85100 The left loss:  29.2201    The right loss:  50.7961
The iteration:  85101 The left loss:  0.271603    The right loss:  0.360422

**Comment by xghc1991** [comment]
can you speak chinese?

**Comment by wuleibupt** [comment]
Can you speak Chinese?

**Comment by LinHungShi** [comment]
@xghc1991 yes, I can. How did you set hyper-parameters?

**Comment by xue1liu2** [comment]
after you training it on kitti data ,the result  may be not good.

**Comment by guandog** [comment]
@xghc1991 could you share the code of your network? I could not make the loss less than 20

**Comment by LinHungShi** [comment]
I updated the network and train the model with driving_finalpass dataset (contains 4400 stereo images) with one epoch. The mean absolute error is around 7.
